### PR TITLE
Fix merge blunder. Improve app:initialize_data command

### PR DESCRIPTION
**Pull request by amagdas**
Fix templates unique name checking.

### CODE REVIEW

**Comment by amagdas** [comment]
@sivakuna-aap @akintolga @marwoodandrew @mdhaman Review pls :bow:

**Comment by akintolga** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -497,49 +501,57 @@
...
-                            }
-                            _.each(template, function(value, key) {
-                                if (!_.isEmpty(value)) {
-                                    if (key !== 'body_html') {
-                                        $scope.origItem[key] = value;
-                                    }
-                                }
+                function resolveDestinations() {
+                    if ($scope.origItem.destination_groups && $scope.origItem.destination_groups.length) {
+                        console.log($scope.origItem.destination_groups);
```
Comment (mdhaman) [inline client/app/scripts/superdesk-authoring/authoring.js]: :+1: 
just remove the console.log

**Comment by marwoodandrew** [comment]
:+1:

**Pull request merged**